Separate experiments

luiztauffer opened the pull request:
Separates jaeger's project into sub-projects, or "experiments".
This improves organization, since we have independent experiments with independent datasets.
Command line shortcut can still be used with an extra argument, e.g.:

```shell
nwbn-gui-jaeger lisu
nwbn-gui-jaeger miao
```

luiztauffer committed:
first commit

luiztauffer committed:
clean up and some fixes to lisu

luiztauffer committed:
clean up and some fiyes for miao

luiztauffer committed:
add load_intan scripts

luiztauffer committed:
datachunkiterator for lisu

bendichter commented:
In general the refactoring looks good, but I don't really like having the names of the experimenters in the import path. The experimenter can change while the data stays the same, so the two shouldn't be linked in the code. Having them as default values in the yaml is fine.

luiztauffer committed:
stores conversion functions in resources, which can be accessed by any experiment-specific modules

luiztauffer committed:
- changes in load_intan modules, supress prints, fix linter errors, block notch filter
- adds list of ElectroGroups as defined by metadata
- path to rhd files as a directory path

luiztauffer committed:
- electrodes info file as an argument

luiztauffer committed:
add behavioral trials for treadmill experiment

luiztauffer committed:
adds all behavioral time series for treadmill experiment

luiztauffer commented:
- individual experiments (experimenters) can be included later as directories containing their own metafile and conversion script (each experimenter uses a different combination of data to be converted)
- the actual data conversion happen in importable modules stored in resources dir

luiztauffer committed:
some cleaning and linter fix

luiztauffer committed:
adds subject metadata to Li Su experiment

luiztauffer committed:
call conversion from command line

luiztauffer committed:
update readme

luiztauffer committed:
- gif
- update readme
